packages/astro/src/vite-plugin-build-css/index.ts

```diff
@@ -124,6 +124,8 @@ export function rollupPluginAstroBuildCSS(options: PluginOptions): VitePlugin {
         }
       }
 
+      if (!chunkCSS) return null; // don’t output empty .css files
```

Comment:
Lastly, this just prevents empty CSS from being omitted. This wasn’t possible before as `vite-plugin-build-html` would assume all chunks existed; now with safer checking there we can simply skip emitting empty chunks without issue.